packages/react/slider/src/Slider.tsx

```diff
@@ -92,56 +92,38 @@ const Slider = React.forwardRef((props, forwardedRef) => {
   const [values = [], setValues] = useControllableState({
     prop: value,
     defaultProp: defaultValue,
-    onChange: onValueChange,
+    onChange: (value) => {
+      const thumbs = [...thumbRefs.current];
+      thumbs[valueIndexToChangeRef.current]?.focus();
```

Comment:
Isn't this an issue if someone is controlling the slider from another input programatically? focus would jump to it right?

Reply:
Ah yeah... hmmm... good point. Let me rethink this.

Reply:
Ah, no this isn't a problem because this only fires when `setValue` is called internally. If the value is changed externally we don't call `setValue`, you'd end up in a loop otherwise.

Reply:
Ah yes because we took care of that in our `useControllableState` hook, that's right! Awesome then!